### ISSUE TITLE
flipper rubygems are not recognized as possible updates

### ISSUE THREAD

**Issue by eduardoj** [depfu/depfu#67]
I our Gemfile, we require the rubygems flipper, flipper-active_record and flipper-ui. See: https://github.com/openSUSE/open-build-service/blob/69304ac39211d687e8ea367ad4289aee0fe3a51c/src/api/Gemfile#L58-L61

Those gems have been upgraded from version 0.28.0 to version 0.28.1.

However, Depfu is not recognizing the upgrade. This is a screenshot of a section of our Depfu dashboard:

![image](https://github.com/depfu/feedback/assets/24919/0b67ff2f-9ee2-4bf8-bc4e-8a25593821a0)

Running `bundle outdated` results in the flipper gems being listed, among others:
```
> bundle outdated                                                                               
Fetching gem metadata from https://rubygems.org/..........                                                                                                                                    
Resolving dependencies....                                                                                                                                                                    
                                                                                                                                                                                              
Gem                    Current  Latest  Requested  Groups                                                                                                                                     
airbrake               9.4.3    13.0.3  >= 0       default                                                                                                                                    
airbrake-ruby          4.6.0    6.2.1   >= 0       default                                                                                                                                    
flipper                0.28.0   0.28.1  >= 0       default                                                                                                                                    
flipper-active_record  0.28.0   0.28.1  >= 0       default                                                                                                                                    
flipper-ui             0.28.0   0.28.1  >= 0       default                                                                                                                                    
jquery-ui-rails        4.2.1    6.0.1   ~> 4.2.1   assets                                                                                                                                     
pundit                 2.3.0    2.3.1   >= 0       default                                                                                                                                    
rack                   2.2.7    3.0.8                                                                                                                                                         
rdoc                   6.3.3    6.5.0   >= 0       development, production                                                                                                                    
selenium-webdriver     4.5.0    4.10.0  >= 0       development, test                                                                                                                          
strscan                3.0.1    3.0.6   = 3.0.1    default                                                                                                                                    
> bundle outdated | grep flipper                                                                                                                            
```

See also: https://github.com/openSUSE/open-build-service/pull/14657

**Comment by eduardoj** [comment]
This is what Depfu shows after merging the pull request:

![Screenshot from 2023-07-18 16-11-04](https://github.com/depfu/feedback/assets/24919/9230c718-c49d-45f9-9d11-cae97b645600)

:thinking:

**Comment by theflow** [comment]
Hi @eduardoj

We always wait 24 hours before a new version shows up in our system to avoid creating unnecessary PRs and in this case flipper also released the `0.28.2` shortly after, which means we wait another 24 hours. You can read more about it here:

https://depfu.com/blog/when-do-we-create-pull-requests


Cheers,
Florian

**Comment by eduardoj** [comment]
I just had to wait. flipper rubygem updates came through Depfu, as inferred from your response. They are already applied. It looks fine now.

Thanks for your time, and continue doing your fantastic work!
